contracts/src/BackedCommunityTokenDescriptorV1.sol

```diff
@@ -3,20 +3,317 @@ pragma solidity 0.8.12;
 
 import {IBackedCommunityTokenDescriptorV1} from "./interfaces/IBackedCommunityTokenDescriptorV1.sol";
 
+import {IBackedBunnyTraitRenderer} from "./traits/IBackedBunnyTraitRenderer.sol";
+import "./utils/Strings.sol";
+
 contract BackedCommunityTokenDescriptorV1 is IBackedCommunityTokenDescriptorV1 {
     uint256 constant activityCategoryId = 0;
-    uint256 constant contributorCateogryId = 1;
+    uint256 constant contributorCategoryId = 1;
     uint256 constant communityCategoryId = 2;
-    uint256 constant communityCallsCategoryId = 3;
-    uint256 constant loansReceivedCateogryId = 4;
-    uint256 constant loansGivenCateogryId = 5;
-    uint256 constant pullRequestsCategoryId = 6;
 
     function tokenURI(
         address owner,
         uint256[] memory scores,
         address specialTraitAddress
     ) external view override returns (string memory) {
-        return "";
+        bool hasSpecialTrait = specialTraitAddress != address(0);
+        string memory traitSVG = hasSpecialTrait
+            ? IBackedBunnyTraitRenderer(specialTraitAddress).renderTrait()
+            : "";
+        string memory traitName = hasSpecialTrait
+            ? IBackedBunnyTraitRenderer(specialTraitAddress).traitName()
+            : "No trait"; // TODO ask joe about default trait placeholder name
```

Comment:
@jringenberg any thoughts here on what should be displayed if user has no trait activated? I was thinking "No trait" or maybe leave blank?

Reply:
Let's say "Backed Community Member" as a default state, a helpful label when it's first been minted with no activity. On the subject of no-trait-activated, this has me thinking we should let people with traits select "no trait" and go back to plain bunn.